### ISSUE TITLE
runtime error when importing `identitystore`

### ISSUE THREAD

**Issue by marafa-sugarcrm** [GoogleCloudPlatform/terraformer#1832]
i keep getting this runtime error when importing identitystore. the command used was `time terraformer import aws --profile=dev --regions=us-east-1 --resources="config"`
```
2024/01/04 08:37:25 aws importing... identitystore
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/GoogleCloudPlatform/terraformer/providers/aws.(*IdentityStoreGenerator).GetIdentityStoreId(0x14001849098?)
	github.com/GoogleCloudPlatform/terraformer/providers/aws/identitystore.go:44 +0x198
github.com/GoogleCloudPlatform/terraformer/providers/aws.(*IdentityStoreGenerator).InitResources(0x0?)
	github.com/GoogleCloudPlatform/terraformer/providers/aws/identitystore.go:169 +0x20
github.com/GoogleCloudPlatform/terraformer/providers/aws.(*AwsFacade).InitResources(0x14003e17cc0?)
	github.com/GoogleCloudPlatform/terraformer/providers/aws/aws_facade.go:61 +0x28
github.com/GoogleCloudPlatform/terraformer/cmd.initServiceResources({_, _}, {_, _}, {{0x140007cd800, 0x56, 0x80}, {0x111d956f0, 0x0, 0x0}, ...}, ...)
	github.com/GoogleCloudPlatform/terraformer/cmd/import.go:209 +0x1a4
github.com/GoogleCloudPlatform/terraformer/cmd.initAllServicesResources(_, {{0x140007cd800, 0x56, 0x80}, {0x111d956f0, 0x0, 0x0}, {0x1084860ee, 0x1e}, {0x1083dd607, ...}, ...}, ...)
	github.com/GoogleCloudPlatform/terraformer/cmd/import.go:166 +0x134
github.com/GoogleCloudPlatform/terraformer/cmd.Import({_, _}, {{0x140007cd800, 0x56, 0x80}, {0x111d956f0, 0x0, 0x0}, {0x1084860ee, 0x1e}, ...}, ...)
	github.com/GoogleCloudPlatform/terraformer/cmd/import.go:98 +0x1a0
github.com/GoogleCloudPlatform/terraformer/cmd.importRegionResources({{0x140008d7740, 0x1, 0x1}, {0x111d956f0, 0x0, 0x0}, {0x1084860ee, 0x1e}, {0x1083dd607, 0x9}, ...}, ...)
	github.com/GoogleCloudPlatform/terraformer/cmd/provider_cmd_aws.go:121 +0x200
github.com/GoogleCloudPlatform/terraformer/cmd.newCmdAwsImporter.func1(0x14000cfe580?, {0x1083cebea?, 0x3?, 0x3?})
	github.com/GoogleCloudPlatform/terraformer/cmd/provider_cmd_aws.go:58 +0x804
github.com/spf13/cobra.(*Command).execute(0x14000cfe580, {0x14001682690, 0x3, 0x3})
	github.com/spf13/cobra@v1.1.1/command.go:850 +0x4c4
github.com/spf13/cobra.(*Command).ExecuteC(0x14000ed0580)
	github.com/spf13/cobra@v1.1.1/command.go:958 +0x34c
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.1.1/command.go:895
github.com/GoogleCloudPlatform/terraformer/cmd.Execute()
	github.com/GoogleCloudPlatform/terraformer/cmd/root.go:36 +0x20
main.main()
	github.com/GoogleCloudPlatform/terraformer/main.go:39 +0x54
```

**Comment by Asgaros** [comment]
Same issue here. Not sure if it is relevant, but I am using version `5.33.0` of the AWS provider for Windows (AMD64).

Used command:
`terraformer plan aws --output hcl --resources="*" --verbose`

**Comment by jeffrey-aguilera** [comment]
Same issue with `version = "~> 5.47"`

**Comment by samveen** [comment]
Probably introduced in #1685  ?

**Comment by samveen** [comment]
Still an issue

**Comment by samveen** [comment]
Fixed in #1808 . However no new release since the fix made it into the codebase
